### PR TITLE
Add needles for gdm_session_switch

### PR DESCRIPTION
**Pull request by chuchingkai**
Testing Result:
http://147.2.212.239/tests/767

openSUSE has shipped SLE-Classic since Leap 42.2, this case will test
gdm session switch among sle-classic, gnome-classic, icewm and gnome.

see also: os-autoinst-distri-opensuse PR#1814
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1814

### CODE REVIEW

File: gdm_session_switch-system-indicator-20160913.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "type": "match"
+    }
+  ],
+  "tags": [
+    "ENV-DESKTOP-gnome",
+    "ENV-DISTRI-opensuse",
+    "ENV-INSTLANG-en_US",
+    "system-indicator"
+  ]
+}
```
Comment (okurz) [inline gdm_session_switch-system-indicator-20160913.json]: I don't know what this needle is for

File: gdm_session_switch-system-indicator-classic-20160913.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      "height": 23
+    }
+  ],
+  "tags": [
+    "ENV-DESKTOP-gnome-classic",
+    "ENV-DISTRI-opensuse",
+    "ENV-INSTLANG-en_US",
+    "system-indicator"
+  ]
+}
```
Comment (okurz) [inline gdm_session_switch-system-indicator-classic-20160913.json]: I don't see how this needle should resemble a "system-indicator"

File: gdm_session_switch-system-indicator-20160913.json (hunk 1 above)
Comment (chuchingkai) [inline gdm_session_switch-system-indicator-20160913.json]: This needle is for 'assert_and_click' the power button to log out

File: gdm_session_switch-system-indicator-classic-20160913.json (hunk 2 above)
Comment (chuchingkai) [inline gdm_session_switch-system-indicator-classic-20160913.json]: My local server has a low performance so nautilus is closed with a delay.
needle 'system-indicator' is only a small area in the top right corner of the desktop, the nautilus window is harmless.

**Comment by chuchingkai** [comment]
some of needles need update on openqa.opensuse.org
https://openqa.opensuse.org/tests/263840#step/gdm_session_switch/13

Can I have the needle editing permission on o.o.o , if not I will update the needle in another PR.

**Comment by okurz** [comment]
for needle editing permissions I recommend to ask o3 admins in [#opensuse-factory](irc://chat.freenode.net/opensuse-factory) if you don't get an answer here.

for https://openqa.opensuse.org/tests/263840#step/gdm_session_switch/13 especially I recommend to make the needle more robust by not relying on the desktop icons. An explicit PR with a reference to a local test run does not harm, though.

**Pull request merged**